Gemfile

```diff
@@ -92,6 +92,7 @@ group :development, :test do
   gem 'rspec_junit_formatter'
   gem 'rspec-rails', '~> 5.0'
   gem 'rspec-sonarqube-formatter'
+  gem 'simplecov'
```

Comment:
We can move this to the `:test` group and add `require: false`, so it won't be required everytimg bundler starts.

Reply:
Done - I havent a clue really, I just copied this from Get Into Teaching :)

Reply:
Leaving it there wouldn't cause any issues, but it's just a bit more efficient now :)